### PR TITLE
New article "WebView2 end-user FAQ"

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR replaces https://github.com/MicrosoftDocs/edge-developer/pull/2521 (Add initial WebView2 power user FAQ), which will be closed w/o merging.

Rendered article for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/end-user-faq?branch=pr-en-us-2524 [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/end-user-faq/microsoft-edge/webview2/concepts/end-user-faq.md)]

AB#43978488

### CODE REVIEW

File: microsoft-edge/webview2/concepts/end-user-faq.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+<!-- ====================================================================== -->
+## What is WebView2?
+
+WebView2 is a way for app developers to embed web content (such as HTML, JavaScript, and CSS) in Windows applications. By including the WebView2 control in an app, a developer can write code for a website or web app, and then reuse that web code in their Windows application, saving time and effort.  See [Introduction to Microsoft Edge WebView2](../index.md).
+
+
+<!-- ====================================================================== -->
+## What's the purpose of the WebView2 executable?
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/end-user-faq.md:23]: Corp-level style guide is flexible: per Dev of orig. PR, the audience for this article is power users, ie technical audience, thus 'executable' is allowed by the corp-level style guide:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/executable-exe - Use _executable_ file only in content for a technical audience. For a general audience, use _program file_ instead.

File: microsoft-edge/webview2/concepts/end-user-faq.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/30/2023
+---
+# WebView2 end-user FAQ
+
+This FAQ is for end users, to explain the `msedgewebview2.exe` application on your computer.  If your question isn't listed here, you can click the **Feedback** button in the top-right corner, to enter a question as an [Issue in the edge-developer Docs repo](https://github.com/MicrosoftDocs/edge-developer/issues), and your question might be added to this article.  See also [Contact the WebView2 Team](../contact.md).
```
Comment (captainbrosset) [inline microsoft-edge/webview2/concepts/end-user-faq.md:13]: Should we explain FAQ at least once at the top of this article? 
Maybe: "This FAQ (Frequently Asked Questions) is for end users, to explain ..."

File: microsoft-edge/webview2/concepts/end-user-faq.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+WebView2 follows the process model that's used in the Chromium browser engine of Microsoft Edge, described in [Process Model and Site Isolation](https://chromium.googlesource.com/chromium/src/+/main/docs/process_model_and_site_isolation.md).  Having functionality broken up into multiple processes help reliability, security, and performance.  Each process performs a certain responsibility, and can do that work uninterrupted, even if one of the other processes hits a problem.
+
+There are normally a few processes:
+*  The WebView2 manager.
+*  A GPU process.
+*  Utility processes, such as network or audio, depending on the content.
+*  Renderer processes.
+
+Your machine will have one set of processes for each app that uses WebView2, and typically one renderer process for each WebView2 control that's in the app, similar to how you have one renderer process per tab in a browser.  For details, see [Process model for WebView2 apps](/microsoft-edge/webview2/concepts/process-model).
```
Comment (captainbrosset) [inline microsoft-edge/webview2/concepts/end-user-faq.md:41]: Consider adding a screenshot of the Task Manager showing an app that uses WebView2, with its process expanded to show the various processes listed above.

File: microsoft-edge/webview2/concepts/end-user-faq.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+
+<!-- ====================================================================== -->
+## What memory and CPU resources does WebView2 use?
+
+In most cases, any extra memory or CPU usage is due to how WebView2 is being used, not WebView2 itself. If the application is rendering non-optimized web content, then the WebView2 control might consume more resources. Task Manager will normally show the processes grouped by parent, but that view is sometimes incorrect if you sort by a column other than **Name**. 
+
+You can also use Microsoft's [Process Explorer](/sysinternals/downloads/process-explorer) tool. If you find that an app is consistently causing heavy resource usage, you can contact Support for that app to report the resource usage.
```
Comment (captainbrosset) [inline microsoft-edge/webview2/concepts/end-user-faq.md:49]:
```suggestion
You can also use Microsoft's [Process Explorer](/sysinternals/downloads/process-explorer) tool. If you find that an app is consistently causing heavy resource usage, contact the support for that app to report the resource usage.
```

**Review by captainbrosset — APPROVED** [review]
Approved with a few minor comments.

File: microsoft-edge/webview2/concepts/end-user-faq.md (hunk 2 above)
Comment (aluhrs13) [inline microsoft-edge/webview2/concepts/end-user-faq.md:13]: Reeza messaged me a good point - We shouldn't point people to GitHub, this is targeted at people who probably don't use GitHub.

**Review by aluhrs13 — APPROVED** [review]

File: microsoft-edge/webview2/concepts/end-user-faq.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/end-user-faq.md:13]: This Frequently Asked Questions (FAQ) page is for end users, to explain the `msedgewebview2.exe` application on your computer.  See also [Contact the WebView2 Team](../contact.md).

File: microsoft-edge/webview2/concepts/end-user-faq.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/end-user-faq.md:41]: @captainbrosset - pls in VS open Win32 GetStart, build, run, capture in Task Manager to replace the pngs below.

@aluhrs13 

gh rendered section:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/end-user-faq/microsoft-edge/webview2/concepts/end-user-faq.md#what-memory-and-cpu-resources-does-webview2-use

Comment (aluhrs13) [reply microsoft-edge/webview2/concepts/end-user-faq.md:41]: We should note with screenshots that Windows 10 and early versions of Windows 11 have pretty mediocre views. New Win11 builds have details.

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/end-user-faq.md:41]: "In Windows 10 and early versions of Windows 11, the **Processes** tab only lists the name of the app that uses WebView2:

In the latest Windows 11 versions, the **Processes** tab lists each WebView2 process:"

**Pull request merged**